lib/index.js

```diff
@@ -109,9 +111,19 @@ module.exports = class Rooftop {
     return post
   }
 
-  _writeTemplates (compiler, template) {
-    const path = template.path
-    const output = template.output
+  _writeTemplates (compiler, options, data) {
+    if (!options.template) { return }
+    const source = options.template.path
+    const output = options.template.output
 
+    node.call(fs.readFile.bind(fs), path.join(source), 'utf8')
+      .then((jadeTpl) => {
+        return W.map(data, (item) => {
+          const renderedTpl = jade.render(jadeTpl, { item: item })
+          const outPath = path.join(output(item))
+          mkdirp.sync(path.dirname(outPath))
+          return node.call(fs.writeFile.bind(fs), outPath, renderedTpl)
+        })
+      })
```

Comment:
I left this comment in an earlier commit, but the best way to write files through a webpack plugin is using `assets` in the emit hook